app/requirements.txt

```diff
@@ -4,6 +4,7 @@ django-filter==0.11.0
 django-extensions==1.6.1
 django-cors-headers==1.0.0
 google-api-python-client==1.5.0
+oauth2client==2.0.1
```

Comment:
Okay. Maybe the `google-api-python-client` requirement adds nothing useful then. The docs confusingly reference the python client library, but then use `oauth2client` in the example, which I expected to be contained or a dependency; I thought I'd simply reprovisioned with the python client line added there.